fbpic/particles/particles.py

```diff
@@ -220,10 +220,9 @@ def dens_func( z, r ) ...
                                         dtype=np.int32 )
             # Register boolean that records if the particles are sorted or not
             self.sorted = False
-        # Register variables when using multithreading
-        self.use_threading = threading_enabled
+
         # Register number of threads
```

Comment:
Note: I also removed `self.use_threading` in the `Particles` class, as it is not used anymore (since the PR in which we removed the old methods using the`weight` function)